redis/cluster.py

```diff
@@ -1114,10 +1098,8 @@ def _execute_command(self, target_node, *args, **kwargs):
                 return response
 
             except (RedisClusterException, BusyLoadingError) as e:
```

Comment:
We can remove this and catch these errors with BaseException as we are not doing anything here